src/itr_evaluator.ml

```diff
@@ -914,7 +914,9 @@ and evaluate_expr (context : 'a context) (e : expr) : record_item =
         | Record_item (Rec_value (Value (Variable _)) as obj)
         | Record_item (Rec_value (Value (MessageValue _)) as obj) ->
           evaluate_expr (Value (ObjectProperty { obj; index; prop }))
-        | _ -> context.get_field event field_path)
+        | _ -> let ans = context.get_field event field_path in
+
+               ans)
```

Comment:
Are these local variables just a leftover artifact of debugging? Think these changes (there are a few similar ones elsewhere) can be reverted